.github/workflows/qa.yaml

```diff
@@ -0,0 +1,36 @@
+name: Quality Assurance
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+permissions:
+  contents: read
+
+jobs:
+  qa:
+    name: Quality Checks
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Git Checkout
+        uses: actions/checkout@v4
+
+      - name: PHP Setup
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: 8.2
+
+      - name: Validate composer.json
+        run: composer validate --strict
+
+      - name: Install dependencies
+        uses: ramsey/composer-install@v2
+
+      - name: Check CS-Fixer
+        run: composer cs:check
+
+#      - name: Check PHPStan
+#        run: composer phpstan
```

Comment:
Is this not working yet, or why is it commented out?

Reply:
correct